src/views/organisationType.njk

```diff
@@ -51,17 +52,36 @@
 
   </form>
 
+  <div id="sorryMessage" style="display: none;">
+    <h1 class="govuk-heading-l">Sorry, you can not access OneLogin GOV.UK from this page </h1>
+    <p class="govuk-body">
+      <a href="/" class="govuk-link"> Go to the GOV.UK OneLogin homepage</a> to find the service you need.
+</p>
+  </div>
+
   <script>
+
     document.addEventListener("DOMContentLoaded", function () {
-      const storedOrganisationType = localStorage.getItem("organisationType");
+      const validJourney = localStorage.getItem("validJourney");
+      console.log(validJourney);
 
-      const form = document.forms.organisationForm
+      if (validJourney === 'true') {
 
-      if (storedOrganisationType) {
-        Array
-          .from(form.elements.organisationType)
-          .find((radio) => radio.value === storedOrganisationType)
-          .checked = true
+        const form = document.forms.organisationForm;
+        const storedOrganisationType = localStorage.getItem("organisationType");
+
+        if (storedOrganisationType) {
+          Array
+            .from(form.elements.organisationType)
+            .find((radio) => radio.value === storedOrganisationType)
+            .checked = true
+        }
+        form.style.display = 'block';
+      } else {
+        document
+          .getElementById('sorryMessage')
```

Comment:
So just looking at the patterns of the other pods, and although this is a good solution we want it to be scalable i think it'd be best that we create a nunjucks file the same way the accounts pod have then render that page if the error response comes back as unauthorised

Reply:
https://github.com/alphagov/di-account-management-frontend/blob/main/src/components/common/errors/404.njk here's an example :D

Reply:
agreed, I initially created it as a file/separate page but then wasn't sure how to render that using the localStorage method so I deleted it :(